compaction/task_manager_module.cc

```diff
@@ -433,6 +434,25 @@ future<> cleanup_keyspace_compaction_task_impl::run() {
     });
 }
 
+future<> global_cleanup_compaction_task_impl::run() {
+    co_await _db.invoke_on_all([&] (replica::database& db) -> future<> {
+        co_await db.flush_all_tables();
+        const auto keyspaces = _db.local().get_non_local_strategy_keyspaces();
+        co_await coroutine::parallel_for_each(keyspaces, [&] (const sstring& ks) -> future<> {
```

Comment:
We should skip keyspaces that do not require/support cleanup (local or use tablets).
See e277ec6aef97eba8b78616ad2fe1245961544f22